README.md

```diff
@@ -37,6 +37,9 @@ const web3 = new Web3();
 
 // Initialize core
 const core = new CirclesCore({
+  gnosisSafeAddress: '0x...',
```

Comment:
Is this the user's safe, or the master safe? Might be helpful to specify in the name, since you'll probably have both floating around

Reply:
Good point! This will be reflected in the `feature/safe` branch.